session_0/word_processing.rb

```diff
@@ -2,13 +2,20 @@
 # any non-lower case alphabet (A..Z) to corresponding lower case
 # alphabet
 def lower_case(words)
-  raise NotImplementedError # TODO
+	wr=[]
```

Comment:
Do you think [map](https://www.rubyguides.com/2018/10/ruby-map-method/) can be used here?

Reply:
I thought map alters the array and each does not. But now I got to know map returns the new array and each returns the original array.